datafusion/physical-optimizer/src/pruning.rs

```diff
@@ -22,12 +22,10 @@
 use std::collections::HashSet;
```

Comment:
`datafusion-expr` dependency is needed by `//! [`Expr`]: crate::prelude::Expr`, should we remove the comment? @alamb

Reply:
In other places, we have changed the link to docs.rs directly. So in this case, change the link to

https://docs.rs/datafusion/latest/datafusion/logical_expr/enum.Expr.html